### ISSUE TITLE
Warning: React does not recognize the asyncOnClick prop on a DOM element.

### ISSUE THREAD

**Issue by rlueder** [abdennour/react-csv#97]
React started throwing a warning in the console, this appears to have started after updating to react-csv 1.0.15, reverting to 1.0.14 fixes the issue (i.e. no warning displayed in console).

> Warning: React does not recognize the `asyncOnClick` prop on a DOM element. If you intentionally want it to appear in the DOM as a custom attribute, spell it as lowercase `asynconclick` instead. If you accidentally passed it from a parent component, remove it from the DOM element.
    in a (created by CSVLink)

```
"react": "^16.4.2",
"react-csv": "^1.0.15"
```

And here's the CSVLink element causing the warning:

```
<CSVLink target="" filename="log_08_28_2018.csv" separator="," uFEFF={true} asyncOnClick={false}>
    <a download="log_08_28_2018.csv" target="" asyncOnClick={false} href="blob:http://localhost:2020/...">
        <span>Export to CSV</span>
    </a>
</CSVLink>
```

**Comment by eliorivero** [comment]
I'm also having this same issue with 1.0.15. As @rlueder mentions, going back to 1.0.14 fixes it.

**Comment by abdennour** [comment]
Thanks @rlueder ! It is now fixed with version: **1.0.15**
